backend/src/Altinn.Authorizationadmin/Altinn.Authorizationadmin/Controllers/DelegationsController.cs

```diff
@@ -82,7 +81,7 @@ public async Task<ActionResult> Post([FromBody] List<PolicyRule> rules)
         /// <response code="500">Internal Server Error</response>
         [HttpPost]
         [Route("authorization/api/v1/[controller]/AddResourceDelegation")]
-        public async Task<ActionResult> AddResourceDelegation([FromBody] List<PolicyRule> rules)
+        public async Task<ActionResult> AddResourceDelegation([FromBody] List<Rule> rules)
```

Comment:
AddResourceDelegation is now just AddRules without Authorization.
We will need a endpoint with user authentication and authorization at some point before frontend can connect.

If we are not also adding the new authentication/authorization for end users together with this change I suggest we remove the endpoint, and just test through AddRules